### PR TITLE
Fix github workflows

### PR DESCRIPTION
**Pull request by zulus**
Test PR, I'm trying to stabilize tests

### CODE REVIEW

File: .github/workflows/windowsPR.yml [hunk 1]
```diff
@@ -24,6 +24,6 @@ jobs:
         maven-version: 3.9.2
     - uses: actions/setup-node@v3
       with:
-        node-version: '16'
+        node-version: '18'
```
Comment (zulus) [inline .github/workflows/windowsPR.yml:27]: Make sure we use same version as in embedder

File: org.eclipse.wildwebdeveloper/package.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
     "yaml-language-server": "1.14.0",
     "vscode-languageserver-protocol": "3.17.4-next.3",
     "vscode-languageserver-types": "3.17.4-next.2",
+    "vscode-languageserver": "9.0.0",
     "vscode-css-languageserver": "file:target/vscode-css-languageserver-1.0.0.tgz",
```
Comment (zulus) [inline org.eclipse.wildwebdeveloper/package.json:13]: After this vscode-languageserver is in root so nested node_modules is not required

File: org.eclipse.wildwebdeveloper/pom.xml [hunk 3]
```diff
@@ -204,6 +204,8 @@
 								<arg>--only=prod</arg>
 								<arg>--no-bin-links</arg>
 								<arg>--ignore-scripts</arg>
+								<arg>--install-strategy=hoisted</arg>
+								<arg>--force</arg>
```
Comment (zulus) [inline org.eclipse.wildwebdeveloper/pom.xml:208]: Make sure dependencies are always fresh, without this tests fail on github but pass jenkins

Comment (zulus) [reply org.eclipse.wildwebdeveloper/pom.xml:208]: Force also will ignore platform requirements, so will be possible to add fsevents for mac users ;)

File: org.eclipse.wildwebdeveloper/package.json (hunk 2 above)
Comment (zulus) [reply org.eclipse.wildwebdeveloper/package.json:13]: I downgraded to 8.2 (9.0 have just 5 days)

**Comment by zulus** [comment]
For now all tests pass, but I'm afraid sooner or later multiple "package.json" will be required to avoid too long resource paths

**Comment by mickaelistria** [comment]
Thank you!

**Comment by vrubezhny** [comment]
Long resource path will highly likely appear until vscode updates its language servers support, see:
- [Iteration Plan for September 2023 #192822](https://github.com/microsoft/vscode/issues/192822)
- [Prepare for 3.17.4 release #1314](https://github.com/microsoft/vscode-languageserver-node/issues/1314)

**Comment by zulus** [comment]
I hope. For now yaml and angular embed and use vscode-languageserver 7.0.0 (protocol 3.16.0), vue rely on vscode-languageserver 8.1.0

**Pull request merged**